source/documentation/using_the_api/getting_updates.md

```diff
@@ -1,20 +1,97 @@
 ## Getting updates
 
-You can find the latest entry number by looking at the register information (use the [`GET /register` endpoint](#getregister)) and comparing the most recent entry number with your own copy.
+You can use [the `GET /download-register/` endpoint](#download) to download a full new copy of a register.
```

Comment:
This endpoint as the first option makes me grinch. Has anything changed that makes us want to offer this option as the most important way to get updates?